src/routes/user-routes.ts

```diff
@@ -0,0 +1,28 @@
+import Elysia, { t } from "elysia";
+import { user } from "../controllers/userController";
+import { Set } from "../utils/utils";
+
+const userBodyCheck = {
+  body: t.Object({
+    id: t.Number(),
+    name: t.String(),
+    email: t.String(),
+    password: t.String(),
+  }),
+};
+
+export const userRoutes = (app: Elysia) =>
+  app.group("/users", (group) =>
+    group
+      .get("/:id", ({ params: { id }, set }) =>
```

Comment:
This will send 500 error if I send request with an `id` that isn’t a number. It should return 400 with text “id is not a number” or "id should be a number"